design/javamp-deprecation.md

```diff
@@ -0,0 +1,115 @@
+# Java Microprofile Application Stack Deprecation
+
+## Key Concepts / Background
+- Kabanero hasn't had any guidelines for deprecating an application stack, so this design will describe a best practice for deprecation.
+
+- The Java Microprofile community has refactored the Java cloud native  application stack to provide many improvements -- delivered as a new stack: the Open Liberty application stack.  The Open Liberty application stack has already been featured in Kabanero.  New Java cloud native applications should now leverage the Java Open Liberty application stack.
+
+- The Java Microprofile application stack maintained in Kabanero is derived from the similar stack at Appsody.  Kabanero will inherit the stack metadata which marks the stack deprecated, as well as the container changes which will print deprecated messages as part of the application start up.  This is a best practice.  The deprecation marker and start up messages indicates to any user of the stack that they should start making plans to change technologies.
```

Comment:
Fine..as long as you don't think "inherit" implies something more specific than "copy/merge from the fork" in this context.

Reply:
Yeah, copy/merge is what was intended.